aregion.cpp

```diff
@@ -1954,8 +1960,12 @@ int ARegion::MoveCost(int movetype, ARegion *fromRegion, int dir, AString *road)
 	int cost = 1;
 	if (Globals->WEATHER_EXISTS) {
 		cost = 2;
-		if (weather == W_BLIZZARD) return 10;
-		if (weather == W_NORMAL || clearskies) cost = 1;
+		if (weather == W_NORMAL || clearskies) {
+			cost = 1;
+		}
+	}
+	if (weather == W_BLIZZARD && !clearskies) {
```

Comment:
I think this is fine, though if this code was ever part of a game that had WEATHER_EXISTS true and did something with blizzard this would break.  A game global would be safer and allow it to coexist.